src/lua/env.lua

```diff
@@ -7,6 +7,7 @@ ffi.cdef[[
 
     int   setenv(const char *name, const char *value, int overwrite);
     int   unsetenv(const char *name);
+    char * getenv_safe(const char *name, size_t buf_size, bool *exceeded);
```

Comment:
Nit: redundant space before `getenv_safe`.

Reply:
This code is removed now, but thanks for noticing.